x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -4,14 +4,16 @@
  * you may not use this file except in compliance with the Elastic License.
```

Comment:
:cool: Thank you for _not_ changing the name of process_event_dot . I have an attachment to the name. It's a nice reminder that everything starts as a dot.

Reply:
To @kqualters-elastic point, we could probably improve it in the future by making it _more_ like a dot and less like a giant thing with a bunch of code in it but :shrug: